From 44d5ed4f1e398188bad5283fd75b91594de54cd0 Mon Sep 17 00:00:00 2001
From: "Jason St. John" <jstjohn@purdue.edu>
Date: Tue, 17 Dec 2013 18:48:43 -0500
Subject: [PATCH] man: be more explicit about option arguments that take
 128-bit IDs in journalctl(1)

It may not be immediately obvious to the reader what "ID128" is, so replace the
example option argument "ID128" with "128-bit-ID".
---
 man/journalctl.xml | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/man/journalctl.xml b/man/journalctl.xml
index dd8cb18..2ee1647 100644
--- a/man/journalctl.xml
+++ b/man/journalctl.xml
@@ -714,7 +714,7 @@
 
                         <varlistentry>
                                 <term><option>--list-catalog
-                                <optional><replaceable>ID128...</replaceable></optional>
+                                <optional><replaceable>128-bit-ID...</replaceable></optional>
                                 </option></term>
 
                                 <listitem><para>List the contents of
@@ -723,7 +723,7 @@
                                 description strings.</para>
 
                                 <para>If any
-                                <replaceable>ID128</replaceable>s are
+                                <replaceable>128-bit-ID</replaceable>s are
                                 specified, only those entries are shown.
                                 </para>
                                 </listitem>
@@ -731,7 +731,7 @@
 
                         <varlistentry>
                                 <term><option>--dump-catalog
-                                <optional><replaceable>ID128...</replaceable></optional>
+                                <optional><replaceable>128-bit-ID...</replaceable></optional>
                                 </option></term>
 
                                 <listitem><para>Show the contents of
@@ -742,7 +742,7 @@
                                 files.</para>
 
                                 <para>If any
-                                <replaceable>ID128</replaceable>s are
+                                <replaceable>128-bit-ID</replaceable>s are
                                 specified, only those entries are shown.
                                 </para>
                                 </listitem>
